account_move_delivery_invoice/reports/delivery_invoice_templates.xml

```diff
@@ -269,6 +269,18 @@
                             </div>
                         </div>
                     </div>
+                    <div t-if="o.secondary_partner">
+                        <strong>Secondary Partner: </strong>
+                        <span t-esc="o.secondary_partner" />
+                    </div>
```

Comment:
I thought we were not supposed to show the secondary partner in the report, but I might be wrong.